AtMap/AtSpecMATMap.cxx

```diff
@@ -279,8 +283,29 @@ XYPoint AtSpecMATMap::CalcPadCenter(Int_t PadRef)
       return {-9999, -9999};
    }
 
-   auto x = AtPadCoord[PadRef][3][0];
-   auto y = AtPadCoord[PadRef][3][1];
+   double x, y;
```

Comment:
x and y need to be given an initial value. Following the convention probably -9999

Reply:
Ok, I added the initialization for x & y.